WORKSPACE

```diff
@@ -36,13 +36,11 @@ bind(
 #
 # Determine SHA256 `wget https://github.com/envoyproxy/envoy/archive/COMMIT.tar.gz && sha256sum COMMIT.tar.gz`
 ENVOY_SHA = "b3be5713f2100ab5c40316e73ce34581245bd26a"
-ENVOY_SHA256 = "79629284ae143d66b873c08883dc6382fac2e8ed45f6f3521f7e7282b6650216"
```

Comment:
Don't remove sha256 here and below? You don't need to change this file actually

Reply:
This sds_api branch was used to demo and test SDS when Envoy SDS work was under review. @quanjielin  is going to use this branch for debugging and testing sds integration with istio.

Reply:
ah ok I thought this is release branch or master.